### ISSUE TITLE
Debian base no longer has glibc?

### ISSUE THREAD

**Issue by danthegoodman1** [GoogleContainerTools/distroless#1340]
**Describe the bug**
```
/app/tangia: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /app/tangia)
/app/tangia: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /app/tangia)
```

We use `gcr.io/distroless/base`.

**To Reproduce**

```
FROM golang:1.20.5 as build

WORKDIR /app

COPY go.* /app/

RUN --mount=type=cache,target=/go/pkg/mod \
    --mount=type=cache,target=/root/.cache/go-build \
    --mount=type=ssh \
    go mod download

COPY . .

RUN --mount=type=cache,target=/go/pkg/mod \
    --mount=type=cache,target=/root/.cache/go-build \
    go build $GO_ARGS -o /app/tangia

# Need glibc
FROM gcr.io/distroless/base

ENTRYPOINT ["/app/tangia"]
COPY --from=build /app/tangia /app/
```

**Expected behavior**
glibc is found and there

**Additional context**

This only started happening today, releases before today worked fine.

**Comment by danthegoodman1** [comment]
it has something to do with go `1.20.5`, go `1.20.4` works fine...

**Comment by mweibel** [comment]
thanks @danthegoodman1 for providing the fix info - indeed after downgrading to 1.20.4 the build works again. 
For reference, the error I had was:
```
/dlv: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /dlv)
/dlv: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /dlv)
```
